src/current/cockroachcloud/cockroachdb-advanced-on-azure.md

```diff
@@ -12,13 +12,6 @@ This page provides information about CockroachDB {{ site.data.products.dedicated
 
 CockroachDB {{ site.data.products.dedicated }} clusters on Azure have the following temporary limitations. To express interest or request more information about a given limitation, contact your Cockroach Labs account team. For more details, refer to the [FAQs](#faqs).
 
-### Editing and scaling
-
-- A cluster must have at minimum three nodes. A multi-region cluster must have at minimum three nodes per region. Single-node clusters are not supported.
```

Comment:
Since single-node will continue to be supported in GCP & AWS, maybe let's keep this.

Reply:
@mdlinville thoughts on this one?

Reply:
This page is about Azure, and Azure does not allow single-node. clusters. I suppose I should put some portion of this back, but not the heading.